storage/runtime/pom.xml

```diff
@@ -21,20 +21,11 @@
         <dependency>
             <groupId>com.google.cloud</groupId>
             <artifactId>google-cloud-storage</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>commons-logging</groupId>
-                    <artifactId>commons-logging</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>javax.annotation</groupId>
-                    <artifactId>javax.annotation-api</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.checkerframework</groupId>
-                    <artifactId>checker-qual</artifactId>
-                </exclusion>
-            </exclusions>
```

Comment:
Why do you remove these exclusions ?
Quarkus relies on jboss-logmanager so we need to exclude commons-loggin, a "compatibility module" is already added from quarkus core.
For javax-annotation Quarkus already includes the corresponding artifact from the Jakarta namespace.
For the check-qual, it's not used at runtime, so we used to exclude it to lower the size of the final artifact.

Reply:
Done. Sorry - I originally removed them just out of curiosity to see if the integration tests would still work; should have kept them there.